info-compiler/src/main/java/com/google/sps/webcrawler/NewsContentProcessor.java

```diff
@@ -19,11 +19,12 @@
 
 /** Static utilities for processing textual content, such as abbreviation. */
 public class NewsContentProcessor {
-  private static final int MAX_WORD_COUNT = 100;
+  static final int MAX_WORD_COUNT = 100;
 
   // @TODO [Implement more advanced processing methods.]
   /** Extracts the first {@code MAX_WORD_COUNT} words from the news article content. */
-  public static NewsArticle process(NewsArticle newsArticle) {
+  public static NewsArticle process(NewsArticle originalNewsArticle) {
+    NewsArticle newsArticle = new NewsArticle(originalNewsArticle);
     String rawContent = newsArticle.getContent();
     String[] splitContent = rawContent.split(" ");
```

Comment:
Maybe you could do `newsArticle.getContent().split(" ")` since you don't use `rawContent` after this?